sympy/stats/tests/test_continuous_rv.py

```diff
@@ -707,8 +707,8 @@ def test_lognormal():
     # Right now, only density function and sampling works
     # Test sampling: Only e^mean in sample std of 0
```

Comment:
This comment can be removed, now that standard deviation is no longer 0.

Reply:
Done. Please merge if no more changes are required.